lte/gateway/fabfile.py

```diff
@@ -253,11 +254,16 @@ def integ_test(
     else:
         ansible_setup(gateway_host, "dev", "magma_dev.yml")
         gateway_ip = gateway_host.split('@')[1].split(':')[0]
+    print("bbuild is set to: %s" % build)
 
-    execute(_dist_upgrade)
-    execute(_build_magma)
+    if not build:
+        execute(_dist_upgrade)
+        execute(_build_magma)
+        execute(_start_gateway)
+    else:
+        execute(_from_registry)
```

Comment:
what about reversing the conditions. It is always nicer to read positive conditions

```
    if  useBuildFromRepo:
        execute(_from_registry)
   else
        execute(_dist_upgrade)
        execute(_build_magma)
        execute(_start_gateway)
```

Reply:
Nah I'm not a positive guy sorry